src/com/cloudogu/ces/cesbuildlib/HttpClient.groovy

```diff
@@ -51,23 +51,23 @@ class HttpClient implements Serializable {
     }
 
     protected String getCurlAuthParam() {
-        "-u ${script.env.CURL_USER}:${script.env.CURL_PASSWORD}"
+        "-u '${script.env.CURL_USER}:${script.env.CURL_PASSWORD}'"
     }
 
     private String getCurlCommand(String httpMethod, String url, String contentType, String data) {
         return "curl -i -X ${httpMethod} " +
             (credentials ? "${getCurlAuthParam()} " : '') +
             (contentType ? "-H 'Content-Type: ${contentType}' " : '') +
             (data ? "-d '" + data + "' "  : '') +
-            "${url}"
+            "'${url}'"
```

Comment:
Also check this (and the one below) for injection vulnerabilities and e.g. escape the sequence.